### DISCUSSION TITLE
征集1.5.5及更新版本的预设名单

### DISCUSSION THREAD

**Discussion by zhexiuamoy** [lss233/chatgpt-mirai-qq-bot#153] [Ideas]
目前已经有的预设有“正常”、"猫娘" 、"张维为"、"怼我" 、"夸我" 、"丁真" 、"MOSS" 、"辩手"、"律师" 、"小说家"、"诗人"、"疯子" 、"室内设计师"，欢迎大家提供预设名单，这两天我试着写一些测试一下

**Comment by woltium** [comment]
linux终端、老胡、擅长编程的chatgpt、傲娇、病娇、雌小鬼、抖s抖m🤤🤤🤤

**Comment by HandyLu8581** [comment]
御坂

**Comment by zsanjin-p** [comment]
贴吧，claude吧的能不能考虑❤️
